modules/log-forwarding-modes.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+
+:_content-type: CONCEPT
+[id="log-forwarding-modes_{context}"]
+= Log forwarding modes
+
+There are two log forwarding modes available: legacy mode, and multi log forwarder mode.
+
+[IMPORTANT]
+====
+Only the Vector collector is supported for use with multi log forwarder mode. The Fluentd collector can only be used with the legacy `instance` log forwarder in the `openshift-logging` namespace.
+====
+
+[id="log-forwarding-modes-legacy_{context}"]
+== Legacy mode
+
+In legacy mode, you can only use one log forwarder in your cluster. The `ClusterLogForwarder` resource in this mode must be named `instance`, and must be created in the `openshift-logging` namespace. The `ClusterLogForwarder` resource also requires a corresponding `ClusterLogging` resource, that also must be named `instance`, and must also be created in the `openshift-logging` namespace.
+
+[id="log-forwarding-modes-multi-clf_{context}"]
+== Multi log forwarder mode
+
+Multi log forwarder mode is available in logging 5.8 and newer versions, and provides the following functionality:
+
+* Administrators can control which users are allowed to define log collection and which logs they are allowed to collect.
+* Users who have the required permissions are able to specify additional log collection configurations.
+* Cluster administrators who are migrating from the deprecated Fluentd collector to the Vector collector can deploy a new log forwarder separately from their existing deployment. The existing and new log forwarders can operate simultaneously while workloads are being migrated.
```

Comment:
For ROSA and OSD, the cluster-admin role is not available. Users can have the dedicated-admin role.

Reply:
Updated to just "administrators"